app/services/registered_bookmarkable.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 #
-# Should only be created via the Bookmark.register_bookmarkable
+# Should only be created via the Plugin::Instance#register_bookmarkable
```

Comment:
Do you think we should deprecate original method? Something like

```
class Bookmark
  def self.register_bookmarkable
     Discourse.deprecate("bla")
     Plugin::Instance#register_bookmarkable(...)
  end
end

Reply:
Personally I don't think there is a need to, it was only used in chat and data-explorer and I'm replacing the data-explorer use in the other PR